nexus/tests/integration_tests/unauthorized.rs

```diff
@@ -498,7 +504,10 @@ async fn verify_endpoint(
             if let Some(&AllowedMethod::GetWebsocket) = allowed {
                 request = request.websocket_handshake();
             }
-            let response = request.execute().await.unwrap();
+            let response = request
+                .execute()
+                .await
+                .expect(&format!("Failed making {method} request to {uri}"));
```

Comment:
nit: I don't really care but I think clippy usually complains about this and wants `.unwrap_or_else(|| panic!(...))`

(clippy currently doesn't run on test code, I think, but that's a bug)

Reply:
Done in 254bf902adf67a46b1ce8332888b0e9c7bd90882